index.css

```diff
@@ -69,7 +67,7 @@ body {
     color: red;
 }
 
-#fetch-success, #download-error, #unknown-error, #dimiss-btn {
+#fetch-success, #download-error, #unknown-error, #dimiss-btn, #token-error {
```

Comment:
It's generally a good idea if you're adding a specific style to a lot of specific elements, to instead replace that with a class. So you'd have:

```
.hidden {
  display: none;
}
```

and then you'd apply the class `hidden` on all these elements. That is both less code but also easier to understand because when looking at the HTML it'll be clear which elements are hidden, as opposed to having to remember that these specific ones are hidden from the CSS.

Reply:
Since these are all messages that pop up when a specific action happens, you could also call this class "pop-up-message" or something meaningful like that (and then if you wanted to wrap them all in a background/border etc that'd be easy)